VeganVirus/VeganVirus/Animation.cpp

```diff
@@ -0,0 +1,54 @@
+#include "Animation.h"
+
+Animation::Animation(Draw* draw, double timeBetweenFrames, const std::vector<const wchar_t*> frameImagePaths)
+{
+	_draw = draw;
+	_timeBetweenFrames = timeBetweenFrames;
+	this->reset();
+
+	// load bitmaps
+	for (const wchar_t* framePath : frameImagePaths)
+	{
+		this->_bitmaps.push_back(new Bitmap(framePath));
+	}
+}
+
+Animation::Animation(Draw* draw, double timeBetweenFrames, const std::vector<const wchar_t*> frameImagePaths, double w, double h)
```

Comment:
First of all I would like to see documentation of what the paramters. Second, it preferable if you give more meaningfull names such as `width` instead of `w`

Reply:
I prefered using w, h throughout the code because I have long expressions for calculating collisions and longer names make them hard to read. Also, I think using w, h is clearly understood as width and height.